<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Testsimulator</class>
 <widget class="QMainWindow" name="Testsimulator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="state_label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>state name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>50</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>knowledge test simulator</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Question">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>511</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Question</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_A">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>230</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>RadioButton</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_B">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>260</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>RadioButton</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_C">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>290</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>RadioButton</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_D">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>330</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>RadioButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_submit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>400</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>submit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_analysis">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>390</y>
      <width>331</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>analysis</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trueanswer">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>460</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>true answer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_next">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>460</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>next</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_youranswer">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>400</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Your answer:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
